src/Win32/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,11 @@
+﻿Axe.Windows.Win32.HotkeyModifier
```

Comment:
I'd expect everything in Win32 to be internal, which (I think) removes the need of the txt file

Reply:
I've made the change